src/app/signalen/samenvatting/edit-link.css

```diff
@@ -0,0 +1,7 @@
+.ams-edit-link:visited {
```

Comment:
Something for our backlog too, I think?

Reply:
Probably, yeah. Do we want to make whether an inline link has a different visited style configurable? So you can  turn it on or off with a prop?

Reply:
https://gemeente-amsterdam.atlassian.net/browse/DES-830